tests/pubsub/testgossipsub.nim

```diff
@@ -277,54 +278,6 @@ suite "GossipSub":
 
     await allFuturesThrowing(nodes[0].switch.stop(), nodes[1].switch.stop())
 
-  asyncTest "GossipSub unsub - resub faster than backoff":
```

Comment:
Why has this been removed?

Reply:
Why is the test wrong?

Reply:
Because it doesn't take into account the fanout mechanism. The message is sent via fanout, it doesn't care about backoff period.

Reply:
Please, add the test back.

Reply:
Even if it's not working?